permissions/create_user_udp_data_storage_writer_permissions.sql

```diff
@@ -0,0 +1,7 @@
+GRANT CONNECT ON DATABASE events TO idp_data_storage_writer;
+
+GRANT USAGE ON SCHEMA idp_data TO idp_data_storage_writer;
+GRANT USAGE ON SCHEMA billing TO idp_data_storage_writer;
+GRANT SELECT, INSERT, DELETE ON ALL TABLES IN SCHEMA idp_data TO idp_data_storage_writer;
```

Comment:
DELETE without UPDATE seems a bit odd but whatevs :-)

Reply:
Ah, missing commit ... `UPDATE` is actually needed but hadn't made commit 🙄

Reply:
Now pushed